src/app.js

```diff
@@ -42,6 +42,7 @@ app.use(compress())
 	.use('/', serveStatic(app.get('public')))
 	.use(bodyParser.json())
 	.use(bodyParser.urlencoded({extended: true}))
+	.use(bodyParser.raw({type: () => true, limit: '10mb'}))
```

Comment:
what the impact of this threshold?

Reply:
it allows us to send binary file content to our feather-services. It was only possible to send json or forms before.